lib/workers/branch/index.js

```diff
@@ -175,6 +175,37 @@ async function processBranch(branchConfig, prHourlyLimitReached, packageFiles) {
       );
     }
 
+    let isStableVersionReleaseTS = true;
+    if (config.stabilityThreshold) {
+      const parsedStabilityThreshold = Duration.fromISO(
+        config.stabilityThreshold
+      );
+      if (parsedStabilityThreshold.invalid) {
+        throw new Error(
+          `Stability threshold error: ${parsedStabilityThreshold.invalid.explanation}`
```

Comment:
Let's move this check to part of config validation. We can maybe still leave it here for safety, but we definitely should detect bad durations and raise a config warning issue instead.

Reply:
you're talking about `validateConfig()` in `lib/config/validation.js`? i first put it there in the for loop after #78 the first time but didn't seem to get called so i moved it where it's now.

Reply:
Yes, I was thinking in the validation code. If we don't throw an error until branch processing then it's pretty late.

Reply:
is there anything that even calls that function before anything starts?

Reply:
It should be called after we read in the renovate.json from the repo